extensions/amp-social-share/amp-social-share.md

```diff
@@ -0,0 +1,80 @@
+<!---
+Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+### <a name="amp-social-share"></a> `amp-social-share`
+
+Displays a social share button.
+
+Example:
+```html
+<amp-social-share type="twitter" width="60" height="44">
+  <script type="application/json">
```

Comment:
Due to the nature of the content (i.e. some of the values will be long), I believe that an embedded script + json makes more sense. It seems more readable and any overriding you embed within the element - so it's a common theme. It was also what was suggested on the proposal.